Lakat/extension.json

```diff
@@ -33,6 +33,12 @@
 		"BeforePageDisplay": "LakatHooks",
 		"SkinTemplateNavigation::Universal": "LakatHooks"
 	},
+	"SpecialPages": {
+		"CreateBranch": "MediaWiki\\Extension\\Lakat\\SpecialCreateBranch"
+	},
+	"ExtensionMessagesFiles": {
+		"LakatAlias": "i18n/Lakat.alias.php"
+	},
```

Comment:
What does this do?

Reply:
This adds reference to the translations and alternative spellings of the title of the special page. If it is not provided then you get warnings on top of screen on some pages, e.g. on the page /wiki/Special:SpecialPages where all special pages are listed. Warnings say something like "You should provide alias for special page title". I've tried to use simple localization file for this, but warning didn't disappear without this additional file.

Reply:
Perfect. Thanks :)